binance_smart_chain/dex/trades/insert_dodo.sql

```diff
@@ -236,8 +250,10 @@ WITH rows AS (
         FROM
             dodoex."DSP_evt_DODOSwap"
         WHERE trader <> '\xbe60d4c4250438344bec816ec2dec99925deb4c7'
+            AND evt_block_time >= start_ts
+            AND evt_block_time < end_ts
     ) dexs
-    INNER JOIN bsc.transactions tx
+    LEFT JOIN bsc.transactions tx
```

Comment:
any reason this changed to LEFT join? the code ran as inner join, is there an impact on row counts and null values in select columns with LEFT used?

Reply:
error occurred during verification, then i change to back, please check it for me , thanks

Reply:
yep, looks good, thank you for the patience and working with me to get this integrated!